### ISSUE TITLE
Gamescreen/HUD freezes

### ISSUE THREAD

**Issue by LightFire53** [ParadiseSS13/Paradise#16229]
**Issue Description**:

The main game screen freezes, seemingly at random. You can keep moving and interacting with things, and hover your mouse over items and still identify. Chat will still work, but the actual game screen will no longer update, including if you move and such. As an aside, it doesn't appear that you can talk or anything.

**What did you expect to happen**: 

For the game screen to not freeze/lock up.

**What happened instead**:

The screen locks up, forcing you to reconnect.

**Why is this bad/What are the consequences**:

Being able to play the game properly is probably good.

**Steps to reproduce the problem**:

This happens at random, no apparent cause. For instance, it normally happens once or twice a play session/night, however I had one round where it froze 3 times rapidly.

**When did the problem start happening**:

Unknown start date, but it didn't happen when I was just a regular player, and started almost immediately after I was reinstated as a GA. Unknown if it is tied to admin permission or something else.

**Extra information**:

Had it happen multiple times on round 27449, and once (at the time of writing) on round 17450.

**Comment by AffectedArc07** [comment]
This is almost certainly a BYOND issue and isn't something we can fix.


We don't control the actual rendering code of the engine so we can't do much.


Try updating your GPU drivers. I have noticed that BYOND will sometimes lock the main viewport up when I lock and unlock my PC, which is when windows changes it's window management and GPU resource allocation.


TLDR: We can't do anything about this
